### DISCUSSION TITLE
artemis operator: how to tell broker to use a specific PVC

### DISCUSSION THREAD

**Discussion by lnthai2002** [arkmq-org/arkmq-org-broker-operator#577] [Q&A] [answered]
Hello,

We have an artemis broker 2.19 running in kubernetes in stand alone mode, this broker uses a persistent volume claim named "data" and mounted to /var/lib/artemis/data. Now that we are moving to artemis operator, we want to mount this PVC to the new broker deployed by the operator. How can we do that? The ActiveMqArtemis CRD only has storage size and storageClassName.
Here is the deploymentPlan section of my chart:

deploymentPlan:
  size: {{ .Values.replicas }}
  requireLogin: true
  messageMigration: true
  persistenceEnabled: true
  podSecurityContext:
    fsGroup: 0
  storage:
    size: {{ .Values.persistence.size }}
    storageClassName: {{ .Values.persistence.storageClass }}
  resources:
    requests:
      cpu: {{ .Values.resources.requests.cpu }}
      memory: {{ .Values.resources.requests.memory }}

**Comment by brusdev** [comment, marked as answer]
The safest way to transfer data is to export and import your data using artemis cli, i.e. `artemis data exp`. If it doesn't work for you could create an issue to extend the ExtraMounts feature to support persistent volumes too.

**Comment by lnthai2002** [comment]
Thank you, i'll try the cli
